app/ts/ScenarioSelection/getScenarioLocation.ts

```diff
@@ -0,0 +1,7 @@
+import _ from 'lodash';
+
+export default function getScenarioLocation(id: string) {
```

Comment:
should have a test. one advantage to not putting these functions in their own file is that it's slightly harder to forget to update/add tests